docs/usage/faq.md

```diff
@@ -127,54 +127,9 @@ See the dedicated [Private npm module support](./getting-started/private-package
 
 ### Control Renovate's schedule
 
-Renovate itself will run as often as its administrator has configured it (e.g. hourly, daily, etc).
-You may want to update certain repositories less often.
-Or you may even want to use different schedules for specific packages.
+To learn all about controlling Renovate schedule, read the [key concepts, scheduling](https://docs.renovatebot.com/key-concepts/scheduling/) docs.
```

Comment:
@rarkins I'm planning to remove the scheduling text and incorporate it in the key concepts page instead. Is that what you want?